sources/Algo/include/Algo.h

```diff
@@ -154,17 +169,19 @@ struct GeneratorDefinition {
    * @param qmax maximum reactive power for the generator
    * @param pmin minimum active power for the generator
    * @param pmax maximum active power for the generator
+   * @param regulatingBusId the Bus Id this generator is regulating
```

Comment:
I'd call it regulatedBusId, to avoid confusion

Reply:
Done